res/values/strings.xml

```diff
@@ -572,24 +572,24 @@
-    <string name="perm_explain_access_to_storage_denied">Delta Chat requires the Storage permission in order to attach or export photos, videos, or audio, but it has been permanently denied. Please continue to the app settings menu, select \"Permissions\", and enable \"Storage\".</string>
-    <string name="perm_explain_access_to_location_denied">Delta Chat requires Location permission in order to attach a location, but it has been permanently denied. Please continue to the app settings menu, select \"Permissions\", and enable \"Location\".</string>
-    <string name="perm_enable_bg_reminder_title">Tap here to receive messages while Delta Chat is in background.</string>
-    <string name="perm_enable_bg_reminder_text">Delta Chat uses minimal resources and takes care not to drain your battery.</string>
+    <string name="perm_explain_need_for_camera_access">To capture photos and video, grant Delta Chat camera access.</string>
+    <string name="perm_explain_need_for_mic_access">To send audio messages, grant Delta Chat microphone access.</string>
+    <string name="perm_explain_need_for_storage_access">Grant Delta Chat access to your files in order to export files or to create backups.</string>
+    <string name="perm_explain_need_for_storage_access_share">Grant Delta Chat access to your files to share them.</string>
+    <string name="perm_explain_access_to_camera_denied">To take photos or shoot videos, first go to the app settings menu, select \"Permissions\", and turn on \"Camera\".</string>
+    <string name="perm_explain_access_to_mic_denied">To send audio messages, first go to the app settings menu, select \"Permissions\", and turn on \"Microphone\".</string>
+    <string name="perm_explain_access_to_storage_denied">To attach or export photos, videos, or audio, first go to the app settings menu, select \"Permissions\", and turn on \"Storage\".</string>
+    <string name="perm_explain_access_to_location_denied">To attach a location, first go to the app settings menu, select \"Permissions\", and turn on \"Location\".</string>
+    <string name="perm_enable_bg_reminder_title">Tap here to receive messages while Delta Chat is in the background.</string>
+    <string name="perm_enable_bg_reminder_text">Delta Chat uses few resources and takes care not to drain your battery.</string>
 
     <!-- dc_str_* resources -->
     <string name="encrypted_message">Encrypted message</string>
 
     <!-- desktop specific strings -->
     <string name="welcome_desktop">Welcome to Delta Chat</string>
     <string name="login_known_accounts_title_desktop">Known accounts</string>
-    <string name="global_menu_preferences_language_desktop">Choose Language...</string>
+    <string name="global_menu_preferences_language_desktop">Choose Language…</string>
```

Comment:
```suggestion
    <string name="global_menu_preferences_language_desktop">Language</string>
```

Reply:
i think this is correct,  it's duplicated `pref_language`, however, we can fix that later ;)